pandapower/converter/powermodels/from_pm.py

```diff
@@ -48,6 +51,24 @@ def read_pm_results_to_net(net, ppc, ppci, result_pm):
         raise OPFNotConverged("PowerModels.jl OPF not converged")   
 
 
+def add_storage_results(net, result_pmi):
+    if "storage" in result_pmi:
+        df = net.res_storage
+        df[["ps", "qs", "se", "qsc"]] = pd.DataFrame([[np.nan, np.nan, np.nan, np.nan]], index=df.index)
+        df[["sc", "sc_on", "sd", "sd_on"]] = pd.DataFrame([[np.nan, np.nan, np.nan, np.nan]], index=df.index)
+    pm_idx = [str(i) for i in range(1, len(net.storage)+1, 1)]
+    pd_idx = net.storage.index.tolist()
+    for m, d in zip(pd_idx, pm_idx):
```

Comment:
iterating throw storages is very slow for grids with many storages. can you vectorize?